src/caf/distribute/gravity_model/multi_area.py

```diff
@@ -427,21 +530,24 @@ def calibrate(
         `caf.distribute.furness.doubly_constrained_furness()`
         `scipy.optimize.least_squares()`
         """
-        for dist in self.dists:
+        for dist in distributions:
             self.cost_function.validate_params(dist.function_params)
         self._validate_running_log(running_log_path)
         self._initialise_internal_params()
         return self._calibrate(  # type: ignore
+            distributions,
+            running_log_path,
             *args,
-            running_log_path=running_log_path,
             **kwargs,
         )
 
     def _jacobian_function(
         self,
         init_params: list[float],
         cost_distributions: list[MultiCostDistribution],
```

Comment:
This type annotation should is now just the class not a list
```suggestion
        cost_distributions: MultiCostDistribution,
```